source/query-bar.txt

```diff
@@ -69,7 +69,7 @@ as in the following filter example:
    { "_id": { "$oid": "58eee4f43d1a10a4e5339aac" } }
 
 When you have entered a valid MongoDB query, the :guilabel:`Filter`
```

Comment:
This paragraph duplicates the one starting on line 60. I don't know if the Filter issue needs to be called out twice.